terraform/development/main.tf

```diff
@@ -165,7 +165,7 @@ module "networking" {
 
   prefix                                  = local.prefix
   vpc_cidr_block                          = "10.0.0.0/16"
-  vpc_flow_cloudwatch_log_expiration_days = 90
+  vpc_flow_cloudwatch_log_expiration_days = 60

```

Comment:
Bit hard to tell what the right number is, but 60 days felt long enough without being too long.